.github/workflows/push-image.yml

```diff
@@ -4,54 +4,107 @@ on:
   release:
     types:
     - published
+
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version of the stack to push'
+        required: false
+
 env:
-  REGISTRIES_FILENAME: "registries.json"
+  REGISTRIES_FILEPATH: "registries.json"
+  STACKS_FILEPATH: "stacks/images.json"
 
 jobs:
   preparation:
     name: Preparation
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
Suggest we do this in separate PR as discussed, same comment below as well.

Reply:
Based on the below comment i'll use 22 instead of latest